src/ws/cockpitauth.c

```diff
@@ -516,6 +516,20 @@ application_parse_host (const gchar *application)
     return NULL;
 }
 
+static gchar *
+application_cookie_name (const gchar *application)
+{
+  const gchar *host = application_parse_host (application);
+  gchar *cookie_name = NULL;
+
+  if (host)
+      cookie_name = g_strdup_printf ("machine-cockpit+%s", host);
```

Comment:
I purposely switched it to avoid potential collisions with other potential application names. This way there is never a chance of it colliding with a non-machine application. But WDYT?